app/controllers/movies_controller.rb

```diff
@@ -10,12 +10,26 @@ def index
   end
 
   def search
-    search = params[:search]
+    @search = params[:search]
     conn = Faraday.new(url: "https://api.themoviedb.org") do |faraday|
       faraday.headers["API-Key"] = 'c74a8c939b809c17fcffb22fc6e5fd03'
     end
-    response = conn.get("/3/search/movie?api_key=c74a8c939b809c17fcffb22fc6e5fd03&query=#{search}")
+    response = conn.get("/3/search/movie?api_key=c74a8c939b809c17fcffb22fc6e5fd03&query=#{@search}")
     json = JSON.parse(response.body, symbolize_names: true)
     @films = json[:results]
   end
+
+  def show
+    conn = Faraday.new(url: "https://api.themoviedb.org") do |faraday|
+      faraday.headers["API-Key"] = 'c74a8c939b809c17fcffb22fc6e5fd03'
+    end
+    response = conn.get("/3/movie/#{params[:id]}?api_key=c74a8c939b809c17fcffb22fc6e5fd03")
```

Comment:
We'll need to add the environmental variable instead of the actual api_key here!

Reply:
Yes, if it is flagged as a conflict, definitely choose your work that is already in Dev